modules/strelka/1.0/config/default.yaml

```diff
@@ -0,0 +1,36 @@
+lcr-modules:
+    
+    strelka:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id} {pair_status}
+            sample_bam: null  # UPDATE
+            sample_bai: null  # UPDATE
+            candidate_indel: null  # UPDATE
```

Comment:
Could you replace the `null  # UPDATE` with `"__UPDATE__"`? I will be submitting a PR this weekend, which has moved to this new method of identifying values that need to be updated.